feature-utils/poly-look/src/react-components/overlays/sideSheet.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { PolyButton, IconButton } from "../buttons";
+
+import "./sideSheet.css";
+
+/**
+ *
+ * Generic container for HTRT content with close button(s).
+ * Meant to be used in combination with SideSwiper.
+ * @param {Object} props
+ * @param {node} [props.children] - HTRT content.
+ * @param {string} [props.okLabel] - Label used for the close button.
+ * @param {callback} [props.onClose] - onClick event handler for the
+ * close buttons.
+ * @param {string} [props.title = ""] - Optional title.
+ * Defaults to empty string;
+ * @param {React.CSSProperties} [props.style] - Additional styles for
+ * the component.
+ * Defaults to empty object;
```

Comment:
You probably don't need to tell this as long as you include the default value (applies to above too)